package.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>fastrtps</name>
+  <version>2.0.0</version>
+  <description>
+     C++ implementation of the DDS standard of the OMG.
```

Comment:
Review this with the description in the README

Reply:
I think the readme description is too long, but I keep it as this description as well